### PR TITLE
fix(storefront):fix product Images don't change on click in IE11

### PR DESCRIPTION
**Pull request by bc-alexsaiannyi**
#### What?

@BC-tymurbiedukhin @yurytut1993 
this PR fixes issue with viewing a Product with multiple images and clicking on one of its images in Internet Explorer 11 , it appears that image doesn't change.

#### Tickets / Documentation

- [BCTHEME-140](https://jira.bigcommerce.com/browse/BCTHEME-140)

#### Screenshots (if appropriate)

Attach images or add image links here.
[IE11 no images changes before and after fix video](https://github.com/bigcommerce/cornerstone/files/4989688/IE11.no.images.changes.before_after.fix.zip)

### CODE REVIEW

File: assets/js/theme/product/image-gallery.js [hunk 1]
```diff
@@ -74,6 +76,18 @@ export default class ImageGallery {
...
+        if (isBrowserIE) {
+            const fallbackStylesIE = {
+                'background-image': `url(${this.currentImage.mainImageUrl}&ampimbypass=on)`,
+                'background-position': 'center',
+                'background-repeat': 'no-repeat',
+                'background-origin': 'content-box',
+                'background-size': 'contain',
+            };
+
+            this.$mainImageNested.css(fallbackStylesIE);
```
Comment (BC-tymurbiedukhin) [inline assets/js/theme/product/image-gallery.js:89]: Could we use css class instead of js obj?

Comment (bc-alexsaiannyi) [reply assets/js/theme/product/image-gallery.js:89]: good point, but we need to change bg-image url depending on chosen image in gallery plus these styles are inline and bad suited for rewriting via css class.

**Comment by bookernath** [comment]
Fixes https://github.com/bigcommerce/cornerstone/issues/1705

**Comment by chippleh1392** [comment]
Thanks for addressing this. Does this fix also update the href of the anchor tag? Also as you're probably aware, the easy zoom functionality does not work on ie11 because of what I assume to be an issue with the third party plugin. Thanks!

**Comment by bc-alexsaiannyi** [comment]
Thanks, Cristofer. This fix works with nested image only. As I see anchor href is being updated with correct URL according to selected image. The difference only is that it uses higher resolution so I decided don’t touch it. I was not sure what else I can affect with it.

**Pull request merged**